db/src/db.rs

```diff
@@ -434,7 +435,7 @@ fn read_ident_map(conn: &rusqlite::Connection) -> Result<IdentMap> {
             bail!(ErrorKind::NotYetImplemented(format!("bad idents materialized view: expected :db/ident but got {}", a)));
         }
         if let TypedValue::Keyword(keyword) = typed_value {
-            Ok((keyword, e))
+            Ok((keyword.as_ref().clone(), e))
```

Comment:
We know that the keys of the map are unique, so there's no value in having `IdentMap` use `Rc`.